ArduPlane/quadplane.cpp

```diff
@@ -1478,6 +1478,9 @@ bool QuadPlane::should_assist(float aspeed, bool have_airspeed)
        (!option_is_set(QuadPlane::OPTION::DISABLE_SYNTHETIC_AIRSPEED_ASSIST) || ahrs.using_airspeed_sensor())) {
         in_angle_assist = false;
         angle_error_start_ms = 0;
+        if (!in_transition() && aspeed > plane.aparm.airspeed_min * 0.5) {
```

Comment:
Not really a fan of this threshold. Using `in_transition` means it needs testing on tailsitters too.